components/camera/camera.go

```diff
@@ -171,6 +160,49 @@ func DecodeImageFromCamera(ctx context.Context, mimeType string, extra map[strin
 	return img, nil
 }
 
+// GetImageFromGetImages is a utility function to quickly implement GetImage from an already-implemented GetImages method.
+// It returns a byte slice and ImageMetadata, which is the same response signature as the Image method.
+// We use the mimeType arg to specify how to encode the bytes returned from GetImages, returning the first image in the response slice.
+func GetImageFromGetImages(ctx context.Context, mimeType string, cam Camera) ([]byte, ImageMetadata, error) {
+	images, _, err := cam.Images(ctx)
```

Comment:
note a todo to pass through the extra field when https://viam.atlassian.net/browse/RSDK-10991 lands